test/menagerie/libs.test.ts

```
@@ -0,0 +1,99 @@
+    },
+    15_000
+  );
+
+  it.skipIf(!hasPythonModule('numpy'))(
+    'records numpy scalar, int64, and NaN behavior',
+    async () => {
+      const bridge = makeBridge();
+      try {
+        await expect(
+          bridge.call('fixtures.library_torture', 'numpy_adversarial', [])
+        ).rejects.toThrow(/NaN|Infinity|serialize|ndarray/i);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)